hindsight/analysis/payload_size.lua

```diff
@@ -0,0 +1,52 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Extract submission sizes and counts for pipeline messages, emitting small
```

Comment:
Please update to the Hindsight markdown docs so we can auto document the pipeline code.

See https://github.com/mozilla-services/lua_sandbox/blob/master/sandboxes/heka/analysis/throughput.lua for a simple template.